src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -1012,9 +1013,48 @@ class spell_hun_tnt : public SpellScriptLoader
         }
 };
 
+// -186257 - Aspect of the cheetah
+class spell_hun_aspect_cheetah : public SpellScriptLoader
+{
+    public:
+        spell_hun_aspect_cheetah() : SpellScriptLoader("spell_hun_aspect_cheetah") { }
+
+        class spell_hun_aspect_cheetah_AuraScript : public AuraScript
+        {
+
+            PrepareAuraScript(spell_hun_aspect_cheetah_AuraScript);
+
+            bool Validate(SpellInfo const* /*spellInfo*/) override
+            {
+                return ValidateSpellInfo
+                ({
+                    SPELL_HUNTER_ASPECT_CHEETAH_SLOW
+                });
+            }
+
+            void HandleOnRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+            {
+                Unit *target = GetTarget();
+                target->CastSpell(target, SPELL_HUNTER_ASPECT_CHEETAH_SLOW, true);
```

Comment:
I can't test it either on retail, but I asked a friend about what happens in that case and he told me that the second aura does not trigger which is the option that makes more sense. So I'm uploading the new code with that constraint.